driver/flambda_backend_flags.ml

```diff
@@ -216,13 +218,15 @@ module Flambda2 = struct
 
   let o2_flags () =
     cse_depth := 2;
+    join_depth := 2;
     join_points := false;
     unbox_along_intra_function_control_flow := true;
     Expert.fallback_inlining_heuristic := false;
     backend_cse_at_toplevel := false
 
   let o3_flags () =
     cse_depth := 2;
+    join_depth := 2;
```

Comment:
Is this supposed to say 2?  The default is set to 5.

Reply:
Ah, that's an oversight. I don't remember why I added this line, so I'll remove it.